app/Resources/views/embed/code.html.twig

```diff
@@ -0,0 +1,20 @@
+{% block content %}
+    {% set languageOptions = ez_field_options('code', 'language') %}
+    {% set language = 'none' %}
+    {% set embedLanguage = ez_field_value(content, 'language') %}
+    {% if languageOptions[embedLanguage] is defined %}
+        {% set language = languageOptions[embedLanguage] %}
```

Comment:
A little bit too much logic here IMO, have you consider placing it in twig extension? Just a question.